templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,157 @@
+                    </a>
+                </li>
+            </ul>
+
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane active" id="registered">
+                    <table class="table table-striped table-condensed">
+                        <thead>
+                        <tr>
+                            <th>&nbsp;</th>
+                            <th>Name</th>
+                            <th>Status</th>
+                            <th colspan="2">&nbsp;</th>
+                            <th>{{ "Priority"|translate }}</th>
+                            <th>{{ "Category"|translate }}</th>
+                            <th>{{ "Nickname"|translate }}</th>
+                            <th>{{ "Access Control"|translate }}</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% for row in registered %}
+                            <tr>
+                                <td class="text-muted"><small>{{ row.id }}</small></td>
+                                <td><strong>{{ row.name }}</strong></td>
+                                <td class="{{ row.statusColor }}">
+                                    <i class="{{ row.statusIcon }}"></i>&nbsp;
+                                    {% if row.status == "Registered" %}
+                                        {{ row.status|translate }}
+                                    {% else %}
+                                        <a class="text-primary" href="./forms_admin.php?id={{ row.id|escape('url') }}&method={{ row.statusMethod|escape('url') }}">{{ row.status|translate }}</a>
+                                    {% endif %}
+                                </td>
+                                <td>{{ row.packageStatus }}</td>
+                                <td class="{{ row.dbStatusClass }}">
+                                    {% if row.dbStatus == "DB Installed" %}
+                                        {{ row.dbStatus|translate }}
+                                    {% else %}
+                                        <i class="fa fa-fw fa-database"></i>&nbsp;<a class="text-danger" href="./forms_admin.php?id={{ row.id|escape('url') }}&method=install_db">{{ row.dbStatus|translate }}</a>
+                                    {% endif %}
+                                </td>
+                                <td><input type='text' size="4" class="form-control input-sm" name='priority_{{ row.id }}' value='{{ row.priority|escape('html_attr') }}'></td>
+                                <td><input type='text' size="10" class="form-control input-sm" name='category_{{ row.id }}' value='{{ row.category|escape('html_attr') }}'></td>
+                                <td><input type='text' size="10" class="form-control input-sm" name='nickname_{{ row.id }}' value='{{ row.nickname|escape('html_attr') }}'></td>
```

Comment:
row.id also needs to get the html_attr
(and as above, are these items now being double escaped by both html and html_attr: more joyous research on escaping)